### ISSUE TITLE
NSwag.EXE is not signed, have to reapprove in Carbon Black after update

### ISSUE THREAD

**Issue by JohnMarkHowell** [RicoSuter/NSwag#3434]
Every time NSwag Studio gets an update and the installer runs, I have to get NSwag.exe unblocked because the exe is not signed. Are there plans to get this exe signed?

**Comment by jeremyVignelles** [comment]
Yes, closing as duplicate of #2906
